lib/strategies/android-chrome.js

```diff
@@ -9,6 +9,30 @@ function AndroidChromeStrategy(options) {
 AndroidChromeStrategy.prototype = Object.create(BaseStrategy.prototype);
 AndroidChromeStrategy.prototype.constructor = AndroidChromeStrategy;
 
+BaseStrategy.prototype._attachListeners = function () {
+  var self = this;
+
+  self.inputElement.addEventListener('keydown', function (event) {
+    if (keyCannotMutateValue(event)) { return; }
+    self._unformatInput(event);
+  });
+  self.inputElement.addEventListener('keypress', function (event) {
+    if (keyCannotMutateValue(event)) { return; }
+    self._unformatInput(event);
+  });
+  self.inputElement.addEventListener('keyup', function (event) {
+    self._reformatInput(event);
+  });
+  self.inputElement.addEventListener('input', function (event) {
+    // Safari AutoFill fires CustomEvents -- mark the input as unformatted without actually unformatting the current value
```

Comment:
I tried Android Chrome with the Google keyboard and the Samsung keyboard. I put a `console.log` inside the `if` block. It was not called. Shall I remove this bit of code, or should we leave it "just in case"?